track_analyzing/track_analyzer/track_analyzer.cpp

```diff
@@ -23,7 +23,17 @@ namespace
     return FLAGS_##name;                                                      \
   }
 
-DEFINE_string_ext(cmd, "", "command: match, info, cpptrack");
+DEFINE_string_ext(cmd, "",
+                  "command:\n"
+                  "match - based on raw logs gathers points to tracks and matches them to "
+                  "features. To use the tool raw logs should be taken from \"trafin\" project "
+                  "production in gz files and extracted.\n"
```

Comment:
production project?
project from production?
project from the production repository?
project from the production servers?

Reply:
production project